src/state.rs

```diff
@@ -12,6 +12,9 @@ use std::rc::Rc;
 use canonical::{Canon, CanonError, Sink, Source, Store};
 use dusk_abi::{HostModule, Query, Transaction};
 use dusk_hamt::Hamt;
+use microkelvin::{
+    BackendCtor, Compound, DiskBackend, PersistError, Persistance, PersistedId,
```

Comment:
Also here, we might start to be a bit future proof adding an alias: `Persistance as Persistence`, and then in the code using `Persistence`.